### PR TITLE
hotfix/JM-7134

### PR DESCRIPTION
**Pull request by ricardofreitasrocha**
### JIRA link (if applicable) ###
https://centralgovernmentcgi.atlassian.net/browse/JM-7134

### Change description ###
- update the update juror options for when a juror has a status of deferred
- update the way we set options for update juror

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by gregs-cgi — APPROVED** [review]

File: client/templates/juror-management/_partials/juror-update-options.njk [hunk 1]
```diff
@@ -13,30 +12,115 @@
 {% set undoFailedToAttend = { value: "undo-failed-to-attend", text: "Undo ‘Failed to attend' juror status" } %}
 {% set deceased = { value: "deceased", text: "Mark as deceased", checked: radioChecked === "deceased", conditional: { html: deceasedHtml } } %}
 
-{% set updateOptions = [deferral, excusal, reassign, postpone, withdraw, undeliverable, deceased, disqualify, responded] %}
+{% set updateOptions = [] %}
 
-{% if jurorStatus === "Responded" and isCourtUser %}
-  {% set updateOptions = [deferral, excusal, reassign, postpone, withdraw, undeliverable, deceased, disqualify, responded, call, transfer, failedToAttend] %}
+{# maybe this can be removed because it would be the default option #}
```
Comment (gregs-cgi) [inline client/templates/juror-management/_partials/juror-update-options.njk:17]: Let's remove it then :D

File: client/templates/juror-management/_partials/juror-update-options.njk [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+{% if jurorStatus === "Responded" and attendances > 0 and onCall and owner !== "400" %}
+  {% set updateOptions = [disqualify, deceased, complete] %}
+{% endif %}
+
+{% if jurorStatus === "FailedToAttend" and owner !== "400" and isSJO %}
+  {% set updateOptions = [undoFailedToAttend] %}
+{% endif %}
+
+{# uncomplete is not implemented through the juror record #}
+{# {% if jurorStatus === "Completed" and owner !== "400" and isSJO %}
```
Comment (gregs-cgi) [inline client/templates/juror-management/_partials/juror-update-options.njk:101]: Is this worth getting rid of it if shouldn't be here then?

Comment (ricardofreitasrocha) [reply client/templates/juror-management/_partials/juror-update-options.njk:101]: you make a good point that I can't say no to :-)

File: client/templates/juror-management/_partials/juror-update-options.njk (hunk 1 above)
Comment (ricardofreitasrocha) [reply client/templates/juror-management/_partials/juror-update-options.njk:17]: I shall do it. thank you

**Pull request merged**